scss/package.json

```diff
@@ -11,7 +11,7 @@
     "sass": "sass test-sin/scss/custom.scss:test-sin/compiled/sin-test-compiled.css",
     "sass-watch": "sass test-sin/scss/custom.scss:test-sin/compiled/sin-test-compiled.css --watch",
     "doc": "sassdoc \"sin-core\" && node ./scripts/update-documentation.js",
-    "doc-herman": "sassdoc \"sin-core\" --theme=herman",
+    "doc-herman": "sassdoc \"sin-core/grid/_simple-grid-mixin.scss\" --theme=herman",
```

Comment:
I made this change just to narrow the scope, so I could show a working sample without going through and fixing every case.

Reply:
This part helped me resolve the issue the most. It never occurred to me that i can narrow down the scope like that. I don't know why ^_^

I started with the same file as you and then started going back one step at a time.